content/source/docs/enterprise/private/automating-the-installer.html.md

```diff
@@ -292,6 +292,13 @@ while ! curl -ksfS --connect-timeout 5 https://tfe.example.com/_health_check; do
 done
 ```
 
+## If the installation does not appear to be configured correctly
+
+If the installation is successful, but prompts for additional configuration rather than skipping that step, it's likely that the supplied configuration files were not applied during installation.
```

Comment:
```suggestion
If the installation script (`install.sh`) exits successfully, but the Replicated web UI prompts for additional configuration rather than skipping that step, it's likely that the supplied configuration files were not applied during installation.
```